emgapi/migrations/0025_auto_20200615_1817.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.24 on 2020-06-15 18:17
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('emgapi', '0024_auto_20200505_1737'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
```

Comment:
Hey Martin, Out of interest, if you create a new table that way, will Django automatically add the new table to the Django model file?

Reply:
It's the other way. We added ```class ChecksumAlgorithm``` and Django created the migration for us.